latest/docs-ref-conceptual/release-notes-azure-cli.md

```diff
@@ -35,7 +35,7 @@ Version 2.0.13
 
 ### Component
 
-* Added deprecation warning to 'az component' commands
+* Deprecated all `component` commands
```

Comment:
This would be the difference between "Deprecated" and "Removed" in a changelog. Deprecated tends to indicate that the commands still exist, but will be replaced or have otherwise been degraded (such as they may not be maintained anymore) whereas "Removed" means they're outright gone.

This means that there's no difference in "will be deprecated" and "deprecated", __unless__ there is an official statement or public policy that deprecation always provides a guarantee of removal within a very specific timeframe once an API is official declared to be deprecated. In that case, I don't think it's appropriate to say "will be deprecated" either, or at least put it in a changelog. The reason for this is users will become aware of the deprecation one of two ways:

* Running the command and getting the warning ("X will be deprecated" -> "The project will make an official changelog announcement that X is deprecated, at which point it goes away in release Y")
* The official changelog saying X is deprecated is released ("X goes away in release Y")